app/assets/javascripts/helpers.js.coffee

```diff
@@ -20,7 +20,14 @@ _.extend PopHealth.Helpers,
     else
       return value
 
-
+  formatNumber: (value) ->
+    return String(value).replace(/\B(?=(\d{3})+(?!\d))/g, ",") unless isNaN(value)
+    
+  formatMRN: (mrn) ->
```

Comment:
Is this a universally valid thing to do? It seems like medical record numbers are truncated after the first dash. Is that something that would apply to all practices?